idea/resources/intentionDescriptions/OmitConstructorKeywordIntention/after.kt.template

```diff
@@ -0,0 +1 @@
+class Foo (x: Int, y: Int)
```

Comment:
Can you please try to remove the space between class name and `(`, otherwise result is not properly formatted.

Reply:
>  if you do `Make public`, it becomes `class Foo (x: Int)`
Oh...

You may choose to fix the issue only for removing constructor intention, as other intentions lay out of scope for this PR. You may report found bug with `Make public` to tracker and decide if you want to fix it too later.

On the other hand if you see some general solution for both intentions I don't mind if it would be a separate commit fixing them at once in this PR.

Reply:
Looks like the other intention's issue is in `KtPrimaryConstructor.addModifier` and `KtPrimaryConstructor.removeModifier` where they remove the constructor keyword. I added a function instead that will remove keyword and space (and I call that in my intention), but I have not changed the behavior in the other intention because that would require updating a lot of other tests. I will file an bug to the tracker in the meantime.